modules/ROOT/pages/clustering/servers.adoc

```diff
@@ -72,11 +83,22 @@ This may not be specified in combination with `allowedDatabases`.
 
 [NOTE]
 ====
-When a server is enabled, if no `OPTIONS` are not provided, the default server values are taken from the settings `initial.server.mode_constraint`, `initial.server.allowed_databases`, `initial.server.denied_databases`, and/or `initial.server.tags`.
+When a server is enabled, if no `OPTIONS` are not provided, the default server's values are taken from the settings:
+
+* xref:configuration/configuration-settings.adoc#config_initial.server.mode_constraint[`initial.server.mode_constraint`]
+* xref:configuration/configuration-settings.adoc#config_initial.server.allowed_databases[`initial.server.allowed_databases`]
+* xref:configuration/configuration-settings.adoc#config_initial.server.denied_databases[`initial.server.denied_databases`]
+* and/or xref:configuration/configuration-settings.adoc#config_initial.server.tags[`initial.server.tags`]
+
+All these settings are only effective on the first startup of the DBMS.
```

Comment:
Because these settings are `initial.server`, not `initial.dbms`, they are only effective when enabling the server.

```suggestion
All these settings are only effective when enabling the relevant server.
```